Userland/Libraries/LibC/syslog.cpp

```diff
@@ -124,7 +124,7 @@ void vsyslog_r(int priority, struct syslog_data* data, char const* message, va_l
         combined.appendff("{}: ", get_syslog_ident(data));
 
     combined.appendvf(message, args);
-    DeprecatedString combined_string = combined.build();
+    DeprecatedString combined_string = combined.to_deprecated_string();
```

Comment:
Make this `auto`?

Reply:
I did a few of these but only looked for "builder.build()". Fixed!